contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -2368,6 +2386,12 @@ contract TypeAndVersion is DualAggregatorBaseTest {
   }
 }
 
+contract Version is DualAggregatorBaseTest {
+  function test_ReturnsTheCorrectVersion() public view {
+    assertEq(s_aggregator.version(), 6, "did not return the right version");
```

Comment:
Why is version 6? Should we change it?

Reply:
It's hardcoded in the contract, we should ask if we need to update it.